src/test/resources/data/typicalDataBundle.json

```diff
@@ -842,6 +842,29 @@
       "isOpeningEmailEnabled": true,
       "isClosingEmailEnabled": true,
       "isPublishedEmailEnabled": true
+    },
+    "publishedSession1InCourse1": {
```

Comment:
Is it possible to reuse `empty.session` instead of adding a new one?

Reply:
What about if we change one of the other open sessions that have publish time way into the future such that publish time follows session opening time?